crates/iota-data-ingestion-core/src/executor.rs

```diff
@@ -23,17 +23,31 @@ use crate::{
 
 pub const MAX_CHECKPOINTS_IN_PROGRESS: usize = 10000;
 
+#[derive(Debug, Clone)]
+pub enum WorkerPoolMsg {
+    /// Send WorkerPool progress status to Executor main loop
```

Comment:
Nit: This is just the message sent, and I think it should have a description independent of the calling scope. E.g. 
```suggestion
    /// Message with information (e.g. `(<task-name>, checkpoint_sequence_number)`) about the ingestion progress. 
```

Reply:
[fixed](https://github.com/iotaledger/iota/pull/4648/commits/f2d9f8da3328c5aae5815778d69daba4cdbd5bc9#diff-dfef1eeaea282ac0a41b60863611b2244808890824a073bf2264ca6fc775a624R30-R58)